lib/govspeak/html_sanitizer.rb

```diff
@@ -40,12 +40,16 @@ def initialize(dirty_html, options = {})
     @allowed_image_hosts = options[:allowed_image_hosts]
   end
 
-  def sanitize
+  def sanitize(relaxed_elements: [])
     transformers = [TableCellTextAlignWhitelister.new]
     if @allowed_image_hosts && @allowed_image_hosts.any?
       transformers << ImageSourceWhitelister.new(@allowed_image_hosts)
     end
-    Sanitize.clean(@dirty_html, Sanitize::Config.merge(sanitize_config, transformers: transformers))
+
+    config = sanitize_config
+    relaxed_elements.each { |el| config[:elements].add(el) }
```

Comment:
I feel like it'd be a bit easier to follow of these elements were added as part of `sanitize_config` given we already have our own elements added in there: https://github.com/alphagov/govspeak/blob/5e514e4d7c2f5b37a793098b9fc6a532d7921ee7/lib/govspeak/html_sanitizer.rb#L58

Perhaps this should be an argument to `sanitize_config`